packages/components/stencil.config.ts

```diff
@@ -9,6 +9,7 @@ export const config: Config = {
   namespace: 'post-components',
   buildDist: true,
   sourceMap: false,
+  globalStyle: 'src/styles/index.scss',
```

Comment:
Is that even necessary? If I remember correctly, this will be applied with JS and thus take no effect.

Reply:
This is the definition of the globalStyle entry file. If we do not declare it, stencil does not know there is a global styles file which should be processed and therefore will not output a global styles file.